L/libgmime/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libgmime"
```

Comment:
https://docs.binarybuilder.org/stable/building/#Name
```suggestion
name = "gmime"
```

Reply:
After you rename this you also need to move the file to a directory with a matching name.

Reply:
Sorry, I don't understand what exactly you mean

Reply:
The file is in `L/libgmime/build_tarballs.jl`, it has to be moved to `G/gmime/build_tarballs.jl`, otherwise jobs will keep failing forever.